iphone/Classes/PlatformModule.m

```diff
@@ -227,6 +231,18 @@ -(NSString*)macaddress
     return [TiUtils appIdentifier];
 }
 
+- (NSString *)identifierForVendor
+{
+    return [[[UIDevice currentDevice] identifierForVendor] UUIDString];
+}
+
+#if defined(USE_TI_PLATFORMIDENTIFIERFORADVERTISING) || defined(USE_TI_PLATFORMGETIDENTIFIERFORADVERTISING)
+- (NSString *)identifierForAdvertising
+{
+    return [[[ASIdentifierManager sharedManager] advertisingIdentifier] UUIDString];
```

Comment:
Apple  [here](https://developer.apple.com/documentation/adsupport/asidentifiermanager) suggest, we should use this by checking isAdvertisingTrackingEnabled. We can expose "isAdvertisingTrackingEnabled" property as well . Ygor Lemos has also suggested the same in comment.